clients/src/test/java/org/apache/kafka/clients/consumer/internals/ShareHeartbeatRequestManagerTest.java

```diff
@@ -80,11 +76,11 @@
 import static org.mockito.Mockito.when;
 
 public class ShareHeartbeatRequestManagerTest {
-    private final long retryBackoffMs = DEFAULT_RETRY_BACKOFF_MS;
-    private final int heartbeatIntervalMs = DEFAULT_HEARTBEAT_INTERVAL_MS;
-    private final int maxPollIntervalMs = DEFAULT_MAX_POLL_INTERVAL_MS;
-    private final long retryBackoffMaxMs = DEFAULT_RETRY_BACKOFF_MAX_MS;
-    private static final String DEFAULT_GROUP_ID = "groupId";
+    private static final int DEFAULT_HEARTBEAT_INTERVAL_MS = 1000;
+    private static final int DEFAULT_MAX_POLL_INTERVAL_MS = 10000;
+    private static final long DEFAULT_RETRY_BACKOFF_MAX_MS = 1000;
+    private static final long DEFAULT_RETRY_BACKOFF_MS = 80;
+    private static final String DEFAULT_GROUP_ID = "group-id";
```

Comment:
For the share groups case, there is a `ShareConsumerTestBuilder` that already contains all these. I think we should use it here

Reply:
Really the pattern established for the consumer tests ought to be adopted for share groups tests too. We are just starting to align them across the board.

Reply:
sure, that makes total sense. My point is simply to keep the share tests using the `ShareConsumerTestBuilder` consistently, and not start partially migrating them as part of this PR.

Reply:
I updated the test to use `ShareConsumerTestBuilder` instead of `ConsumerTestBuilder` now. Let me know if that works, all tests still pass.